net.davidotek.pupgui2.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
was this submitted upstream?

Reply:
There seems to be a bug with AppImageBuilder that the AppStream can't contain a `launchable` tag for some reason.